### ISSUE TITLE
Add support for riot.js tag file 

### ISSUE THREAD

**Issue by silveryiris** [Glavin001/atom-beautify#731]
The tag file is similar to HTML block.

This is the example from riot.js [live demo](http://riotjs.com/play/)

`sample.tag`

```html
<sample>
  <h3>{ message }</h3>
  <ul>
    <li each={ techs }>{ name }</li>
  </ul>

  <script>
    this.message = 'Hello, Riot!'
    this.techs = [
      { name: 'HTML' },
      { name: 'JavaScript' },
      { name: 'CSS' }
    ]
  </script>

  <style scoped>
    :scope { font-size: 2rem }
    h3 { color: #444 }
    ul { color: #999 }
  </style>
</sample>
```

**Comment by prettydiff** [comment]
Could you try running your riot.js code through http://prettydiff.com/?m=beautify

Pretty Diff is already included with atom-beautify.  If you are already satisfied with the results the next trick will be how to associate the tool so that riot.js files are specifically beautified with Pretty Diff, because the default beautifier is js-beautify and this is association is based only upon the file extension.  Clearly riot.js has a `.js` file extension, but is not actually JavaScript code.

**Comment by silveryiris** [comment]
The tag file works good with http://prettydiff.com/ to me.

riot.js has custom file extension `.tag` which will be compilered to native javascript , html and css. 

Hope  I didn't confuse the extension which should be beautify , Thanks .

**Comment by prettydiff** [comment]
So long as the file is supplied the extension `.tag` to clearly separate it from JavaScript files and there is no conflict with anything else using that extension it seems support can be added immediately.  Glavin is busy right now so your options are:

1. Wait for Glavin to add this support as a new language
2. Modify the atom-beautify code to support the new language with Pretty Diff as the default beautifier and submit a pull request.

**Comment by prettydiff** [comment]
I submitted a pull request to provide support for this language.

**Comment by prettydiff** [comment]
The relevant code is merged to master, so now this is just pending the next release.

**Comment by Glavin001** [comment]
Published to v0.28.28
